substrate/client/transaction-pool/tests/integration.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// Testsuite of fatp integration tests.
+
+pub mod zombienet;
+
+use futures::future::join;
+use txtesttool::{
+	execution_log::ExecutionLog,
+	runner::RunnerFactory,
+	scenario::{ScenarioPlanner, SendingScenario},
+	transaction::TransactionRecipe,
+};
+use zombienet::NetworkSpawner;
+use zombienet_sdk::subxt::OnlineClient;
+
+// Test which sends future and ready txs from many accounts
+// to an unlimited pool.
+#[tokio::test(flavor = "multi_thread")]
+async fn send_future_and_then_ready_from_many_accounts() {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+	let net = NetworkSpawner::init_from_asset_hub_fatp_high_pool_limit_spec().await.unwrap();
+	let collator = net.get_node("charlie").unwrap();
```

Comment:
Maybe just node name?
Would be nicely readable?
```suggestion
	let charlie = net.get_node("charlie").unwrap();
```

Reply:
This isn't relevant anymore. Resolving